common/src/main/scala/react/common/style/package.scala

```diff
@@ -1,162 +1,158 @@
-    def isEmpty: Boolean = styles.isEmpty
+  def toJsObject: js.Object = Style.toJsObject(this)
 
-    def nonEmpty: Boolean = styles.nonEmpty
+  def extract[A](key: String)(implicit S: StyleExtractor[A]): Option[A] =
+    S.extract(this, key)
 
-    def toJsObject: js.Object = Style.toJsObject(this)
+  def remove(key: String): Style =
+    if (this.styles.contains(key))
+      Style(this.styles - key)
+    else
+      this
 
-    def extract[A](key: String)(implicit S: StyleExtractor[A]): Option[A] =
-      S.extract(this, key)
+  def when_(pred: => Boolean): Style =
+    if (pred)
+      this
+    else
+      Style.Empty
 
-    def remove(key: String): Style =
-      if (this.styles.contains(key))
-        Style(this.styles - key)
-      else
-        this
+  def unless_(pred: => Boolean): Style =
+    if (pred)
+      Style.Empty
+    else
+      this
 
-    def when_(pred: => Boolean): Style =
-      if (pred)
-        this
-      else
-        Style.Empty
+}
 
-    def unless_(pred: => Boolean): Style =
-      if (pred)
-        Style.Empty
-      else
-        this
+object Style {
 
-  }
+  def toJsObject(style: Style): js.Object =
+    style.styles.toJSDictionary.asInstanceOf[js.Object]
 
-  object Style {
+  def fromJsObject(o: js.Object): Style = {
+    val xDict = o.asInstanceOf[js.Dictionary[String | Int]]
+    val map   = (for ((prop, value) <- xDict) yield prop -> value).toMap
+    Style(map)
+  }
 
-    def toJsObject(style: Style): js.Object =
-      style.styles.toJSDictionary.asInstanceOf[js.Object]
+  val Empty: Style = Style(Map.empty)
 
-    def fromJsObject(o: js.Object): Style = {
-      val xDict = o.asInstanceOf[js.Dictionary[String | Int]]
-      val map   = (for ((prop, value) <- xDict) yield prop -> value).toMap
-      Style(map)
+  given Eq[String | Int] = Eq.instance { (a, b) =>
+    (a: Any, b: Any) match {
+      case (a: String, b: String) => a === b
+      case (a: Int, b: Int)       => a === b
+      case _                      => false
     }
+  }
 
-    val Empty: Style = Style(Map.empty)
+  given Eq[Style]     = Eq.by(_.styles)
+  given Monoid[Style] = new Monoid[Style] {
+    override val empty: Style                       = Style(Map.empty)
+    override def combine(a: Style, b: Style): Style =
+      Style(a.styles ++ b.styles)
+  }
+}
 
-    given Eq[String | Int] = Eq.instance { (a, b) =>
-      (a: Any, b: Any) match {
-        case (a: String, b: String) => a === b
-        case (a: Int, b: Int)       => a === b
-        case _                      => false
-      }
-    }
+/**
+ * type to represent css class
+ */
+opaque type Css = List[String]
 
-    given Eq[Style]     = Eq.by(_.styles)
-    given Monoid[Style] = new Monoid[Style] {
-      override val empty: Style                       = Style(Map.empty)
-      override def combine(a: Style, b: Style): Style =
-        Style(a.styles ++ b.styles)
-    }
+object Css:
+  final class ClassnameCssOps(cn: (js.UndefOr[String], js.UndefOr[Css])) {
```

Comment:
Trying to convert this to a an extension method proved to be a can of worms, with `toJs` colliding with other methods.
I may revisit this once we get rid of sui

Reply:
When we get rid of SUI, we may no longer need this.

Reply:
It is used in a couple of other facades
Maybe I could add an extension with another name (method different than `toJs`) and port only those (except SUI)

Reply:
Actually it seems to work right away, amazingly metals refactoring did handle a rename

Reply:
It appears it did work. Of course, not I can't find your actual changes in the 116 files. 👅